probe_scraper/runner.py

```diff
@@ -431,10 +428,24 @@ def load_glean_metrics(
         metrics = defaultdict(dict)
         pings = defaultdict(dict)
         for repo_name, commits in commits_by_repo.items():
+            this_repo = [r for r in all_repos if r.name == repo_name][0]
+
             for commit, paths in commits.items():
-                tags_files = [p for p in paths if p.name == GLEAN_TAGS_FILENAME]
-                metrics_files = [p for p in paths if p.name == GLEAN_METRICS_FILENAME]
-                pings_files = [p for p in paths if p.name == GLEAN_PINGS_FILENAME]
+                tags_files = [
+                    p
+                    for p in paths
+                    if any([str(p).endswith(fp) for fp in this_repo.tag_file_paths])
+                ]
+                metrics_files = [
+                    p
+                    for p in paths
+                    if any([str(p).endswith(fp) for fp in this_repo.metrics_file_paths])
+                ]
+                pings_files = [
+                    p
+                    for p in paths
+                    if any([str(p).endswith(fp) for fp in this_repo.ping_file_paths])
+                ]
```

Comment:
We recently (#577) started allowing any file name, but we still had this hardcoded here.
We have the file names configured, so IMO we should use that knowledge to look for the right files.
This turned out to be a bit messy because of the double-iteration.